kfs-core/src/main/java/edu/arizona/kfs/fp/document/validation/impl/DisbursementVoucherInvoiceNumberEnteredValidation.java

```diff
@@ -17,7 +17,6 @@
 
     @Override
     public boolean validate(AttributedDocumentEvent event) {
-        boolean superValidate = super.validate(event);

```

Comment:
Correct, this was a mistake when the code was ported, as the super call was not present in KFS 3. I also verified stepping through in a debugger that super's validation were all related to add/update validations, which should not be the case for a review action, as in this case.